nextflow.config

```diff
@@ -165,32 +264,35 @@ env {
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+// Disable process selector warnings by default. Use debug profile to enable warnings.
+nextflow.enable.configProcessNamesValidation = false
+
 def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.tracedir}/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_timeline_${trace_timestamp}.html"
 }
 report {
     enabled = true
-    file    = "${params.tracedir}/execution_report_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_report_${trace_timestamp}.html"
 }
 trace {
     enabled = true
-    file    = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.tracedir}/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
 }
 
 manifest {
     name            = 'nf-core/oncoanalyser'
-    author          = 'Stephen Watts'
+    author          = """Stephen Watts"""
     homePage        = 'https://github.com/nf-core/oncoanalyser'
-    description     = 'A comprehensive cancer WGS/WTS analysis and reporting pipeline'
+    description     = """A comprehensive cancer DNA/RNA analysis and reporting pipeline"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=21.10.3'
-    version         = '1.0dev'
+    nextflowVersion = '!>=22.10.5'
+    version         = '0.3.1'
```

Comment:
Should the version be 1.0.0 for the first release?

Reply:
Yes it should 😁